pennylane/vqe/vqe.py

```diff
@@ -158,20 +158,26 @@ def simplify(self):
         self._ops = ops
 
     def __str__(self):
-        terms = []
+        # lambda function that formats the wires
+        wires_print = lambda ob: ",".join(map(str, ob.wires.tolist()))
+
+        terms_ls = []
 
         for i, obs in enumerate(self.ops):
-            coeff = "({}) [{{}}]".format(self.coeffs[i])
 
             if isinstance(obs, Tensor):
-                obs_strs = ["{}{}".format(OBS_MAP[i.name], i.wires.tolist()[0]) for i in obs.obs]
-                term = " ".join(obs_strs)
+                obs_strs = [
+                    "{}:{}".format(OBS_MAP.get(ob.name, ob.name), wires_print(ob)) for ob in obs.obs
```

Comment:
(Minor suggestion): how about using an f string here? (slightly out of scope to the changes)
```suggestion
                    f"{OBS_MAP.get(ob.name, ob.name)}:{wires_print(ob)}" for ob in obs.obs
```